workflow-templates/executor-cd.yml

```diff
@@ -4,6 +4,9 @@ on:
   push:
     branches:
       - $default-branch
+  release:
+    types:
+      - created
```

Comment:
L24 needs to be adapted as well. When a release is published, `WyriHaximus/github-action-get-previous-tag@v1` can not retrieve the newly-created git tag.

Reply:
Done. Tested on SimpleIndexer with both a PR and a dummy tag release. https://github.com/jina-ai/executor-simpleindexer/actions